azure-servicebus/azure/servicebus/control_client/servicebusservice.py

```diff
@@ -1111,7 +1111,8 @@ def create_event_hub(self, hub_name, hub=None, fail_on_exist=False):
         request = HTTPRequest()
         request.method = 'PUT'
         request.host = self._get_host()
-        request.path = '/' + _str(hub_name) + '?api-version=2014-01'
+        #request.path = '/' + _str(hub_name) + '?api-version=2014-01'
```

Comment:
Is there a reason to leave these comments in if we've moved to the x_ms_version above?